server/index.js

```diff
@@ -143,16 +196,15 @@ app.post('/api/article-review', (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.post('/api/user-review/:newArticleId', (req, res, next) => {
-  const articleId = Number(req.params.newArticleId);
-  const fakeUserId = 1;
+app.post('/api/user-review', (req, res, next) => {
+  const { userId, articleId } = req.query;
```

Comment:
The userId here needs to come from `req.user` now that you have authorization middleware instead of the `req.query` or `req.body`. We don't want to trust the client to tell us the correct user id.